packages/react-union-scripts/cli.js

```diff
@@ -7,7 +7,7 @@ function runCommand(command) {
 
 const command = process.argv[2];
 
-if (['start', 'build', 'test'].includes(command)) {
+if (['start', 'build', 'test', 'start-ssr-server'].includes(command)) {
```

Comment:
The naming of the file does not match (it's kebab-case), but I don't want to refactor react-union-scripts until I get everything working.